manifests/t/Teclib'/GLPIAgent1/7/2/1.7.2/Teclib'.GLPIAgent1.7.2.installer.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Teclib'.GLPIAgent1.7.2

```

```suggestion
PackageIdentifier: Teclib.GLPIAgent
```
Versions don't need to be part of the identifier